src/modules/profile/components/Team.js

```diff
@@ -0,0 +1,15 @@
+import { h, Component } from "preact";
```

Comment:
This can be a stateless component if this component's job is only to render jsx.

Reply:
```
export default const Team = () => ( 
           <div>
                <img className={styles.image} src={this.props.team.logo} />
                <h1 className={styles.name}>{this.props.team.name}</h1>
            </div>
);
 
```